charts/acp/templates/migration.yaml

```diff
@@ -69,7 +69,7 @@ spec:
             - --sql.url
             - {{ (.Values.sql).url }}
             - --config
-            - /data/config.yaml,/data/extraconfig.yaml,/secret/secret.yaml
+            - {{ join "," .Values.migrateJob.config }}
```

Comment:
Does it make sense to move this config to values.yaml?
I would rather keep it here and add sth like extraArgs to values.yaml for custom args. (or extraConfig for custom additional custom configs)

Reply:
The same applies to

```
## Array with container arguments to add to the ACP container
##
args:
  - server
  - start
  - --config
  - /data/config.yaml,/data/extraconfig.yaml,/secret/config.yaml
  - --demo
  - --metrics
  - --create-default-tenant
  - --create-default-workspaces
```

I would rather keep it hardcoded in the templates, and add a way to add extraArgs.

Same for import job

```
importJob:
  ## Enables the import job
  ##
  enabled: false

  ## Config files path
  ##
  config:
    - /data/config.yaml
    - /data/extraconfig.yaml
    - /secret/config.yaml
```

Reply:
https://cloudentity.atlassian.net/browse/DEVOPS-1882